inference-engine/src/gna_plugin/gna2_model_debug_log.cpp

```diff
@@ -161,6 +164,15 @@ void WriteInputAndOutputTextGNAImpl(const Gna2Model & gnaModel, const std::strin
     }
 }
 
+template<class MapType, class KeyType, class ResType>
+static ResType FindInMapOrReturnUnknown(MapType map, KeyType key, ResType res) {
```

Comment:
1. consider `def`/`defaultValue` instead of `res`
2. consider removing `ResType` template parameter and use `typename MapType::mapped_type`

Reply:
Done.